source/MRViewer/MRSceneCache.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+#include "MRMesh/MRMeshFwd.h"
+#include <optional>
+
+namespace MR
+{
+
+// class to cached scene objects data
+class SceneCache
+{
+public:
+    // invalidate all cached data
+    static void invalidateAll();
+
+    // get all selectable object in tree
+    // same as getAllObjectsInTree<Object>( &SceneRoot::get(), ObjectSelectivityType::Selectable )
```

Comment:
looks like it also returns ancillary objects witch is equal to `getAllObjectsInTree<Object>( &SceneRoot::get(), ObjectSelectivityType::Any )`

Reply:
fixed